### PR TITLE
Update n.Makefile

### PR DESCRIPTION
**Pull request by i-like-robots**
And enable coverage task to be run locally without conflict

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,33 +2,33 @@ include n.Makefile
...
 	make unit-test
 endif
 
 run:
-	$(NPM_BIN_ENV); nbt run --local
+	nbt run --local
 
 build-production:
-	$(NPM_BIN_ENV); nbt build --skip-js --skip-sass --skip-hash
+	nbt build --skip-js --skip-sass --skip-hash
```
Comment (matthew-andrews) [inline Makefile]: you can eliminate `build-production` entirely if you `npm install --save-dev haikro` 😄

Comment (i-like-robots) [inline Makefile]: Will that know to skip static assets (or is that not necessary)?

Comment (matthew-andrews) [inline Makefile]: yes

Comment (matthew-andrews) [inline Makefile]: it will know

**Pull request merged**